backend/util/notifications_queue_manager.py

```diff
@@ -0,0 +1,215 @@
+"""Notifications Queue Manager."""
+
+import json
+import time
+from google.appengine.api import taskqueue
+from service_messages import send_message_notification
+
+__all__ = ['NotificationsQueueManager', 'Notification', 'NotificationNIL']
+
+queue = taskqueue.Queue('notifications-manage-pull')
+notification_id = {
```

Comment:
I think that the ids would be more significant if they were the initials of its respective value

Reply:
Placing initials can cause repeated ids.